src/ray/raylet/worker_pool_test.cc

```diff
@@ -1348,6 +1406,119 @@ TEST_F(WorkerPoolTest, PopWorkerWithRuntimeEnv) {
   ASSERT_EQ(worker_pool_->GetProcessSize(), 3);
 }
 
+TEST_F(WorkerPoolTest, RuntimeEnvUriReference) {
+  valid_uris.clear();
```

Comment:
Move this ` valid_uris.clear();` to TearDown?

Reply:
Good for me!